content/docs/user-guide/visualization/animation/animation-editor/ragdoll/ragdoll-adding-to-animation-graph.md

```diff
@@ -0,0 +1,61 @@
+---
+description: ' Adding a ragdoll to an Animation Graph in the Open 3D Engine Animation Editor. '
+title: Adding a Ragdoll to an Animation Graph
+---
+
+When you create an animation graph to control the ragdoll simulation of your character, you do the following:
++ Prepare the actor asset.
```

Comment:
nit: Bulleted list comment again (I won't go on repeating it but a small formatting pass might be worth it, not a huge deal though)

Reply:
Updated